target/ppc/fpu_helper.c

```diff
@@ -3339,3 +3339,189 @@ void helper_xssubqp(CPUPPCState *env, uint32_t opcode,
     *xt = t;
     do_float_check_status(env, GETPC());
 }
+
+/*
+ * This function should be changed before sending the patches,
+ * as it's here as a placeholder since the functions that set
+ * FPSCR also raise an exception.
+ */
+static inline void compute_fp_flags(CPUPPCState *env, uintptr_t retaddr)
```

Comment:
Maybe It'd be good to rewrite it to make it clearer, but the problem is that other functions would raise an exception before setting all FPSCR bits (and possibly wrongfully unsetting FR/FI), so this function seeks to avoid that